src/main/java/software/amazon/awssdk/crt/auth/credentials/Credentials.java

```diff
@@ -15,7 +15,8 @@ public class Credentials {
     private byte[] sessionToken;
 
     /**
-     * Default constructor
```

Comment:
nit: this comment sounds a bit awkward. consider rewriting.
Something similar to following? And also consider adding comment to regular constructor.
Constructs Anonymous Credentials. 
Signer will skip signing request if Anonymous Credentials are passed in.
This method is deprecated and left for backwards compatibility. Prefer createAnonymousCredentials instead.